app/models/node.rb

```diff
@@ -20,6 +20,8 @@ class Node < ApplicationRecord
   enumerize :state, in: STATES, scope: true
   content_attribute :content_body
   content_attribute :name
+  # options is not currently versioned but could be in the future
+  store_attribute :content, :options
```

Comment:
I agree, but I think the way content is versioned is different from how options will be... rather than LCS, we'd probably want to use a structural diff mechanism like https://github.com/benjamine/jsondiffpatch ...